api/info/connect_handler/connect_handler.go

```diff
@@ -0,0 +1,326 @@
+// connect_handler.go
+// This file implements the ConnectRPC handler for the InfoService
+// It should satisfy the infov1.InfoServiceHandler interface generated from service.proto
+
+package connect_handler
```

Comment:
As discussed offline, it's do this in the `info` package in `service.go`. This should make the diff easier to review, and is more aligned with the goal of this project of migrating the existing api rather than removing one and adding a new one.